<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transactionsCardView</class>
 <widget class="QWidget" name="transactionsCardView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="cardIcon">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>65</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="timeStamp_Label">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>timeStamp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="transactionLabel_Label">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Label</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="amount_Label">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+-amount</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
